src/components/organisms/AdminVenueView/components/EventsView/EventsView.scss

```diff
@@ -0,0 +1,108 @@
+@import "scss/constants.scss";
+
+$events-view-margin: 75px 75px 10px;
+
+.EventsView {
+  background-color: $black;
+  position: relative;
+  height: 75%;
+  width: 85%;
+  margin: $events-view-margin;
```

Comment:
Looks like this is used to 'center' the event container in the middle of the screen. You should add `justify-content: center` and `align-items: center` to parent style `Timing__right-content` and remove this margin because this creates a false sense of centered content.